src/olympia/reviewers/templatetags/jinja_helpers.py

```diff
@@ -56,10 +56,10 @@ def version_status(addon, version):
 def reviewer_page_title(context, title=None, addon=None):
     """Wrapper for reviewer page titles.  Eerily similar to dev_page_title."""
     if addon:
-        title = u'%s :: %s' % (title, addon.name)
+        title = u'%s – %s' % (title, addon.name)
     else:
         section = _('Reviewer Tools')
-        title = u'%s :: %s' % (title, section) if title else section
+        title = u'%s – %s' % (title, section) if title else section
```

Comment:
Now addons-server is python3 only we can take the opportunity to remove the `u'...'` prefixes from the strings in this function.